src/sora-academy.md

```diff
@@ -1,3 +1,53 @@
+less centralized, sparking a sector known as Decentralized Finance,
+and with access to a crypto asset wallet, people can participate in
+activities such as swapping tokens (exchanges), liquidity provision,
+NFT Markets, and more. The SORA Academy is the best way to get you started in DeFi on the SORA network
+
+## Decentralization and Finance
+
+When you participate in any financial activity nowadays, such as investing in stocks or bonds, there are several hurdles you must jump through in order to begin trading, your identity must be verified and you must open an account with an intermediary service to trade. Albeit, some of those services provide the option to pool with other users and buy fractions of stocks, instead of the whole stock, as was the norm up until these services were developed.
+
+It used to be that the difference between purchasing a token and stock was that you could buy a fraction of a token, for example buying and owning €1000 of BTC, while a whole BTC token costs €36,771.31 (when this article was written), but you could only buy a whole stock. As aforementioned, there are services that fractionalize stocks now, so this is no longer the case.
+The big difference between Centralized finance and Decentralized
+finance, however, is that in a decentralized platform, you own your
+keys.
+
+This means that all the requirements you need are a wallet and some tokens. If you were to do these activities through a centralized finance crypto platform, you would need to provide personal identification for a process known as KYC (know your customer), which is required for the platform to comply with anti-money laundering (AML) policies.
+After setting up an account, this platform would sometimes give you the facility to purchase tokens using a credit card or other means of payment (with a verified identity), however, you would not own the keys to your account. In other words, your tokens don’t really belong to you, unless you decide to move them from the CeFi app to your wallet (usually paying a transfer fee).
+
+On the contrary, using a DeFi interface, your wallet serves as your identification, and the transactions you make are done directly with your wallet, meaning you own everything you receive and do not have to make any transfers afterward.
+
+_Now you know the difference between both models, but what exactly IS decentralized finance?_
+
+## Defining DeFi
+
+As of 2021, there are [over 6000 crypto tokens in existence](https://www.investing.com/crypto/currencies) across several blockchains. All having different values and functions on different blockchains. Because of the smart contracts mentioned earlier, there are several finance tools (or dApps, i.e., Decentralized Applications) that use them as a foundation. The most important DeFi activities are swapping (exchanging tokens for others), Pooling (adding liquidity to a token pair), and most recently, NFT trading (the digital equivalent of being an art collector or dealer).
```

Comment:
`Because of the smart contracts mentioned earlier` they are not mentioned earlier in this article, what are you referring to? another page? if so, better link there

Reply:
Please check latest commit, I have added the missing info